README.md

```diff
@@ -2,31 +2,359 @@
+"Kansas", it would be difficult to avoid getting companies with "Kansas"
+in the street address or city name.
+For example, this query would produce an incorrect result as it matches every row in the table:
+
+```sql
+SELECT name
+FROM company
+WHERE address LIKE '%Kansas%'
+```
+
+ We need to evolve the
+table to divide the address into multiple columns, each containing a single
+atomic value (street could possibly be further divided, but most applications
+won't require that level of atomicity):
+
+| id  | name           | street         | city        | state         | zip   |
+|-----|----------------|----------------|-------------|---------------|-------|
+| 1   | SuperMart      | 123 Fiction St | Kansas City | Missouri      | 64101 |
+| 2   | BigCo          | 45 Rue Dr      | Kansas City | Kansas        | 66012 |
+| 3   | MegaMart       | 5 Street Blvd  | Topeka      | Kansas        | 66601 |
+| 4   | GiantCo        | 1 Kansas Lane  | Boston      | Massachusetts | 02111 |
+| 5   | 100DollarStore | 45 Mystery Dr  | Seattle     | Washington    | 98101 |
+
+Now we can easily query to get the names of companies in the state of Kansas:
+
+```sql
+SELECT name
+FROM company
+WHERE state = 'Kansas'
+```
+
+The query results in 2 companies:
+
+```text
+BigCo
+MegaMart
+```
+
+Atomicity also means that a column should not contain a list of values.
+Let's look at another example database that contains two tables: `employee`
+and `department`:
+
+
+<table>
+<tr>
+<th>Employee Table</th>
+<th>Department Table</th>
+</tr>
+
+<tr>
+<td>
+
+| id  | first_name | last_name  | salary |
+|-----|------------|------------|--------|
+| 1   | Kiran      | Willow     | 76000  |
+| 2   | Dani       | Elm        | 99000  |
+| 3   | Hao        | Pine       | 45000  |
+| 4   | Tal        | Oak        | 88000  |
+| 5   | Yuri       | Birch      | 150000 |
+
+</td>
+
+<td>
+
+| id  | name      | location   | employees |
+|-----|-----------|------------|-----------|
+| 1   | Payroll   | Building A | 4, 5      |
+| 2   | Marketing | Building B | 1, 2, 3   |
+
+</td>
+</tr>
+
+</table>
+
+
+- The `employee` table stores one row per employee, and each column stores an atomic value.
+- The `department` table stores the name and location, along with a comma-separated
+  list of ids of employees that work for that department. 
+  The string represents a list of values and thus is not atomic.
+
+What if we want to count the number of employees per department?
+It is difficult to write such a query since the `employees` column is not atomic.
+
+## One-To-Many Relationships
+
+The relationship between `department` and `employee` is **one-to-many**:
+- An employee works for one department, while each department may have any number of employees.
+
+We can model the employee and department as separate entities
+as shown in the **Entity Relationship Diagram** (ERD) below.
+The edge represents the **one-to-many** relationship.
```

Comment:
reworded as "connecting line" rather than "edge"?  There should be relationship showing in the ERD. Does it not show?

Reply:
Yes there is! Sorry, when I was reviewing earlier my brain did not translate "edge" into "connecting line" - my bad! But probably more clear to say "connecting line" just in case that throws students too